spec/test-utils/test_indexeddb_cryptostore_dump/full_account/README.md

```diff
@@ -0,0 +1,5 @@
+## Dump of a libolm indexeddb cryptostore to test migration of a full account
+
+It was created by pasting the same code as in the README of the parent directory into the browser console.
+
+The corresponding user data, key query and backup response can be found in `index.ts`
```

Comment:
Rather than repeating this text three times, it might be better just to put something in the top-level README:

> Each directory contains, in `dump.json`, a dump of data created by pasting the following code into the browser console; and in `index.ts`, details of the user, pickle key, and corresponding key query and backup responses.

Then each individual README can just be a one-line description of the data in that directory.